### ISSUE TITLE
Add some oled driver for DYmore 2.24" oled  SSD1309 

### ISSUE THREAD

**Issue by EYITEA** [Ingwie/OpenAVRc_Dev#59]
Hi, can you add this oled driver to the project.  I see it's very good if you add it, oled with a size of 2.24 " will add to the luxury of your project. I'm sure it is. Because we from Indonesia really like the look of oled with a larger size. thank you.

**Comment by Ingwie** [comment]
Hi,

Where could i find this oled documentation ?

Did you read this documentation (page26) : https://github.com/Ingwie/OpenAVRc_Hw/blob/V3/PCB%20Shield/Mega_2560%20core%20mini_full_2.2_PM/ShieldV2.2_bleu.pdf
It explain how to use this kind of screen (switch SPI to I2C, etc ...)

**Comment by testeur68** [comment]
Et pourquoi Ducros se décarcasse ? Merci Christophe. Envoyé depuis mon téléphone Huawei-------- Message original --------De : Bracame ***@***.***>Date : dim. 6 mars 2022 à 12:03À : Ingwie/OpenAVRc_Dev ***@***.***>Cc : Subscribed ***@***.***>Objet : Re: [Ingwie/OpenAVRc_Dev] Add some oled driver for DYmore 2.24" oled  SSD1309  (Issue #59)
Hi,
Where could i find this oled documentation ?
Did you read this documentation (page26) : https://github.com/Ingwie/OpenAVRc_Hw/blob/V3/PCB%20Shield/Mega_2560%20core%20mini_full_2.2_PM/ShieldV2.2_bleu.pdf
It explain how to use this kind of screen (switch SPI to I2C, etc ...)

**Comment by EYITEA** [comment]
okay, thanks for all the help, I'll try it and greetings from Indonesia for all of them.

**Comment by sloped-soarer** [comment]
I use an SSD1309 OLED for my project, but not the DIYMore version. In fact, i thought i gave a DIYMore OLED to @Ingwie.
Green OLED is okay in low light levels, but not so good outdoors. I also noted that some OLED displays put quite a bit of noise on the supply rails.
Just My opinion, best of luck.

**Comment by EYITEA** [comment]
If so what is the impact on radio?  Does it affect the speed of data transmission?  Thank you

**Comment by Ingwie** [comment]
Hi, Hello, Bonjour,
SSD1306 (1309) driver do the same job -> No impact.
Some LCD are faster and alow very low latency on very fast protocols.
I2C drivers "paint" the screen whith several pass to keep the better computation timing.
There is no low speed code in OpenAVRc to give to the pilot the better experience posiible.

**Comment by EYITEA** [comment]
Very good information, I am very happy to meet you all.  I will continue to study to learn more about OpenAVRc.

**Comment by EYITEA** [comment]
> Hi, Hello, Bonjour, SSD1306 (1309) driver do the same job -> No impact. Some LCD are faster and alow very low latency on very fast protocols. I2C drivers "paint" the screen whith several pass to keep the better computation timing. There is no low speed code in OpenAVRc to give to the pilot the better experience posiible.

Hi, I'm planning to implement RTC into my radio, but I'm using an i2c oled LCD type, if I want to implement it simultaneously, is that possible, or can I simply add a multiplexer module so that both can run simultaneously. Thanks.

**Comment by pierrotm777** [comment]
Yes you can use rtc with an i2c screen. But sometime we need to remove the pull-up on sda and scl signals (only if you have some issue).

**Comment by EYITEA** [comment]
Thanks for the information, I'll try it.

**Comment by Ingwie** [comment]
Solved it was a SSH1106
